docs/reference/migration/migrate_6_0/search.asciidoc

```diff
@@ -30,3 +30,34 @@
 
 The search shards API no longer accepts the `type` url parameter, which didn't
 have any effect in previous versions.
+
+==== `docvalue_fields`
+
+The `docvalue_fields` now returns a formatted value for the field. This is
+likely mostly an issue if you used `docvalue_fields` to return dates as a number
+of milliseconds since Epoch. It is still possible to format dates as the number
+of milliseconds since Epoch by using the `epoch_millis` format:
+
+[source,js]
+--------------------------------------------------
+GET _search 
+{
+    "docvalue_fields" : [ { "name": "my_date", "format": "epoch_millis" } ]
```

Comment:
We should keep this as the default if possible in 5.x so we aren't making breaking changes, I think.